wifiphisher/pywifiphisher.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 # pylint: skip-file
+from __future__ import absolute_import
+from __future__ import print_function
 import subprocess
```

Comment:
@cclauss I have a question: the pull request will only include the changes in green/red, right?  The rest of the code looks different from the wifiphisher/master base.

Reply:
@cclauss Another question: Should I rebase the merge of master to this branch as well?  Wouldn't my changes be hard to read?